user-context.md

```diff
@@ -20,6 +20,10 @@ web:
     uri: "/me"
 ```
 
+### Authentication
+
+This endpoint requires authentication, and accepts all forms of authentication. 
```

Comment:
As written, this makes sense to me.  Accept all forms of authentication, as this route will always have a response that is tied to the user context, and all of our authentication means do identify a user.